internal/pkg/workspace/workspace.go

```diff
@@ -104,7 +105,7 @@ func Use(fs afero.Fs) (*Workspace, error) {
 }
 
 // Create creates a new Workspace in the current working directory for appName with summary if it doesn't already exist.
-func Create(appName string, fs afero.Fs) (*Workspace, error) {
+func Create(appName string, fs afero.Fs) (*Workspace, error) { 
```

Comment:
nit: so that `git` or something does not print out annoying warnings against trailing whitespace 😂 
```suggestion
func Create(appName string, fs afero.Fs) (*Workspace, error) {
```